src/50_Samples_%26_Templates/Product_Landing_Page.html

```diff
@@ -0,0 +1,282 @@
+    -moz-border-top-left-radius: 8px;
+    -moz-border-bottom-left-radius: 8px;
+    -webkit-border-top-left-radius: 8px;
+    -webkit-border-bottom-left-radius: 8px;
+    border-top-left-radius: 8px;
+    border-bottom-left-radius: 8px;
+
+    -webkit-transition: all .5s;
+    -moz-transition: all .5s;
+    transition: all .5s;
+  }
+  .header input[type=search]:focus {
+    box-sizing: border-box;
+    width: 320px;
+    height: 36px;
+    background-color: #fff;
+    border-color: #eb407a;
+
+    -webkit-box-shadow: 0 0 5px rgba(109,207,246,.5);
+    -moz-box-shadow: 0 0 5px rgba(109,207,246,.5);
+    box-shadow: 0 0 5px rgba(109,207,246,.5);
+  }
+  .header input:-moz-placeholder,
+  .header input::-webkit-input-placeholder {
+    color: #999;
+  }
+  .header input:focus + #logo {
+    visibility: hidden;
+    opacity: 0;
+    transition: visibility 0s 0.5s, opacity 0.5s linear;
+  }
+  .header input + #logo {
+    visibility: visible;
+    opacity: 1;
+    transition: opacity 1s linear;
+  }
+  /* navigation */
+  .nav a {
+    font-weight: 400;
+    color: white;
+    text-transform: uppercase;
+    margin: 16px;
+    height: 36px; 
+  }
+  ul {
+    height: auto;
+    padding: 8px 0px;
+    margin: 0px;
+  }
+  .nav li { 
+    display: inline; 
+    padding: 0; 
+  }
+  /* deals */
+  .deal {
+    margin-right: 16px;
+    display: block;
+    height: 75px;
+    line-height: 75px;
+    color: #111;
+    padding: 0;
+  }
+  .deals h3 {
+    margin: 16px 0;
+  }
+  .deal.card {
+    background: white;
+  }
+  .deal amp-img {
+    line-height: 24px;
+    font-weight: 400;
+    font-size: 24px;
+    vertical-align: middle;
+    float: left;
+  }
+  .deal:hover {
+    background-color: #ccc;
+  }
+</style>
+
+<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!-- #### Header with Search -->
+  <!-- AMP supports forms, which means you can directly integrate a product search into your AMPs.  
+  
+  AMP doesn't support custom Javascript, but you can use CSS3 animations to enrich your page, like. Try focusing the search box. -->
+  <div class="header">
+    <form  method="post" action="/samples_templates/product_listing/preview/search" target="_top">
```

Comment:
Should we specify that in this example the form is not working, but it would be in the product_listing one?